db/migrate/20210125051022_devise_create_users.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class DeviseCreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ''
+      t.string :encrypted_password, null: false, default: ''
+      t.string :username,           null: false, default: ''
+      t.integer :rating,            null: true
```

Comment:
Rating?

Reply:
Player MMR as proposed by @ChristopherJMiller in discussing user schema.